TS.fsx

```diff
@@ -531,8 +531,25 @@ let EmitDictionaries flavor =
         | "Object" -> Pt.printl "interface %s {" dict.Name
         | _ -> Pt.printl "interface %s extends %s {" dict.Name dict.Extends
 
+        let emitJsonProperty (p: ItemsType.Root) =
+            Pt.printl "%s: %s;" p.Name.Value p.Type.Value
+
+        let removedPropNames = 
+            getRemovedItems ItemKind.Property flavor 
+            |> Array.filter (matchInterface dict.Name)
+            |> Array.map (fun rp -> rp.Name.Value)
```

Comment:
How is `Set.contains` faster than `Array.contains`? My impression is the opposite.